README.md

```diff
@@ -46,39 +29,40 @@ The following dependencies are required and must be installed separately.
 
 ### Install via `go get`
 
-To install Annie, use `go get`, or download the binary file in the [Releases](https://github.com/iawia002/annie/releases) page.
+To install Annie, use `go get`, or download the binary file from [Releases](https://github.com/iawia002/annie/releases) page.
 
 ```bash
 $ go get github.com/iawia002/annie
-...
-$ annie [args] URL
 ```
 
 
-## Get Started
+## Getting Started
 
 ### Download a video
 
 ```console
-$ annie -s 127.0.0.1:1080 https://youtu.be/Gnbch2osEeo
```

Comment:
I prefer just remove `-s 127.0.0.1:1080`, `annie https://youtu.be/Gnbch2osEeo`, do not change others